core/query/evalQuery.mli

```diff
@@ -0,0 +1,9 @@
+open Utility;;
```

Comment:
The double semicolons `;;` are unnecessary.

Reply:
Yes, there are many instances of this in the code. I will remove them when I come across them (like everybody else should).

Reply:
If we really wanted to get rid of `;;` we could add an automated check that complains about them, same as trailing whitespace...  I am sure some of them have been there for over a decade.